paper-tristate-checkbox.html

```diff
@@ -311,40 +305,51 @@
           type: String,
           value: null,
           reflectToAttribute: true,
-          notify: true
+          notify: true,
         },
 
         /**
          * The element's value when `'on'`/checked.
          */
         checked: {
           type: String,
-          value: "x"
+          value: 'x',
         },
 
         /**
          * The element's value when `'off'`/unchecked.
          */
         unchecked: {
           type: String,
-          value: " "
+          value: ' ',
         },
 
         /**
          * The element's value when `null`/indeterminate.
          */
         indeterminate: {
           type: String,
-          value: ""
+          value: '',
         },
 
         /**
          * The element's value based on its current state.
          */
         value: {
           type: String,
-          computed: '_computeValue(state)'
-        }
+          computed: '_computeValue(state)',
+        },
+      },
+      ready() {
+        this._regularTap = this._regularTap.bind(this);
+      },
+      attached() {
+        this.setAttribute('aria-checked', 'mixed');
```

Comment:
Is there a reason this isn't working in the host attributes?

Reply:
Yeah, it's interesting. A behavior dependency has itself another dependency which listeners to changes to an ariaActiveAttribute. As a result, a distant observer is triggered which removes the attribute so it stopped showing up in 2.x. The dependency is PaperInkyFocusBehavior which has a dependency on IronButtonState. This was an interesting bug to tackle.